dashboard/test/lib/services/script_seed_test.rb

```diff
@@ -138,6 +195,8 @@ class ScriptSeedTest < ActiveSupport::TestCase
     expected_counts = original_counts.clone
     expected_counts['LessonGroup'] -= 1
     expected_counts['Lesson'] -= 2
+    expected_counts['LessonActivity'] -= 2
```

Comment:
Might be out of the scope of this PR but is there a test that the correct Lessons/LessonActivities/etc are getting deleted? This part of the code makes me super nervous

Reply:
Great point. There is not. I will think about how to add this in a subsequent PR.

Reply:
I don't think this is going to be quick, since I am having some trouble with deletion tests in a subsequent PR. I'm going to open a Jira task to make sure we don't lose track of this: https://codedotorg.atlassian.net/browse/PLAT-609